vendor/bat-native-ledger/src/bat/ledger/internal/media/vimeo.cc

```diff
@@ -83,13 +85,18 @@ std::string Vimeo::GetIdFromVideoPage(const std::string& data) {
 }
 
 // static
-std::string Vimeo::GenerateFaviconUrl(const std::string& id) {
-  if (id.empty()) {
-    return "";
-  }
+std::string Vimeo::GenerateFaviconUrl(const std::string& data) {
+    if (data.empty()) {
```

Comment:
alignment is off. It should be just 2 spaces

Reply:
Fixed